pkg/dash/dash.go

```diff
@@ -458,3 +503,48 @@ func enableOpenCensus() error {
 
 	return nil
 }
+
+// findKubeConfig looks for kube config from .kube or provided by user
+func findKubeConfig(logger log.Logger, temporaryKubeConfig kubeconfig.TemporaryKubeConfig) <-chan string {
+	r := make(chan string)
+
+	go func() {
```

Comment:
The one location where you call `findKubeConfig` is already a goroutine, you don't need the nested go routine here.